docs/getting-started/devenv/using-visual-studio-code.md

```diff
@@ -1,16 +1,381 @@
+The contents of `global.json` are:
+
+```js
+{
+    projects: [
+        "src",
+        "test"
+    ]
+}
+```
+
+At this point, your directory tree should look like this:
+
+
+```
+/golden
+|__global.json
+|__/src
+   |__/app
+   |__/library
+/test
+   |__/test-library
+```
+
+### Writing the library
+
+Your next task is to create the library. In the terminal window
+(either the embedded terminal in VS code, or another terminal),
+cd to `golden/src/library` and type the command `dotnet new`.
+
+The `dotnet new` command creates a new application project. Because
+you want a library project, you're going to make a few changes to
+the generated `project.json` file.
+
+First, remote the `buildOptions` node:
+
+```js
+"buildOptions" {
+    "emitEntryPoint": true
+}
+```
+
+Next, you'll need to update the dependencies section to create
+a library assembly, rather than an appliction. You'll remove the
+`Microsoft.NETCore.App` node:
+
+```js
+"dependencies": {
+    "Microsoft.NETCore.App": {
+        "type": "platform",
+        "version": "1.0.0-rc2-300702"
+    }
+}
+```
+And replace it with a reference to the .NET Standard library:
+
+```js
+"dependencies": {
+    "NETStandard.Library": "1.5.0-rc2-24027"
+}
+```
+
+You'll also need to update the frameworks section to use 
+.NET Standards instead of `netcoreapp1.0`:
+
+```js
+"frameworks": {
+    "netcoreapp1.0" : {
+        "imports": "dnxcore50"
+    }
+}
+```
+
+becomes:
+
+```js
+"frameworks" : {
+    "netstandard1.5": {
+        "imports": "dnxcore50"
+}
+```
+
+This library will make use of JSON representation of objects, so you'll want to
+add a reference to the `Newtonsoft.Json` NuGet package. In`project.json`
+add the latest pre-release version of the package as a dependency:
+
+```js
+"depdendencies": {
+    "NETStandard.Library": {
+        "version": "1.5.0-rc2-24027"
+    },
+    "Newtonsoft.Json": "9.0.1-beta1"
+}
+```
+
+After you've finished adding those dependencies, you need to install those
+packages into workspace. The `dotnet restore` command updates all dependencies,
+and writes a `project.lock.json` file in the project directory. This
+file contains the full dependency tree of all the dependencies in your
+project. You don't need to read this file, its used by tools in the .NET
+Core SDK.
+
+Now, let's update the C# code. Let's create a `Thing` class that contains
+one public method. This method will return the sum of two numbers,
+but  will do so by converting that number to a JSON string, and then
+deserializing it. Rename the file `Program.cs` to `Thing.cs`. Then, replace
+the existing code (for the template-generated Hello World) with the following:
+
+```cs
+using static Newtonsoft.Json.JsonConvert;
+
+namespace Library
+{
+    public class Thing
+    {
+        public int Get(int left, int right) =>
+            DeserializeObject<int>($"{left + right}");
+    }
+}
+```
+
+This makes use of a number of modern C# features that you can learn
```

Comment:
A comment like this might be more helpful if it mentioned what the modern features are and linked directly to articles about them.